requirements.txt

```diff
@@ -0,0 +1,5 @@
+dnspython
+matplotlib
+ocflib
```

Comment:
I assume this downloads the latest `ocflib` from PyPI and bundles it?

Reply:
Correct